nvflare/lighter/tree_prov.py

```diff
@@ -128,6 +128,7 @@ def package(self, project: Project, ctx: ProvisionContext):
         )
 
         sample_sim_config = {
+            "job_name": "edge_job",
             "endpoint": f"http://localhost:{self.rp_port}",
             "num_devices": 10000,
             "num_active_devices": 100,
```

Comment:
No need for the following any more:
num_active_devices
cycle_duration
device_reuse_rate

Reply:
right, removed them and updated the README